src/test/NativeERC20.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+import {Test} from "forge-std/Test.sol";
+
+pragma solidity ^0.8.0;
+
+contract NativeERC20 is Test {
```

Comment:
Would love a comment above this contract explaining what it does - and that transfer changes balances of native too